packages/amplify-e2e-core/src/utils/rds.ts

```diff
@@ -241,6 +253,57 @@ export const createRDSCluster = async (config: RDSConfig): Promise<ClusterInfo>
   }
 };
 
+/**
+ * Accesses the local RDS Aurora serverless V2 cluster with one DB instance using the given input configuration.
+ * @param config Configuration of the database cluster.
+ * @returns EndPoint address, port and database name of the accessed RDS cluster.
+ */
+export const useRDSCluster = async (config: RDSConfig): Promise<ClusterInfo> => {
+  try {
+    // Send requests to get the cluster and instance config info
+    const databaseName = 'defaultdb';
```

Comment:
Is a hardcoded db name the right way to go here? What happens if multiple tests (e.g., one from a local machine and one from CICD) run in the same account & same region? Wouldn't they overwrite each other?

Reply:
Done - got rid of the hardcoded db name.